roxctl/deployment/check/check_test.go

```diff
@@ -270,14 +273,66 @@ func (d *deployCheckTestSuite) createMockEnvironmentWithConn(conn *grpc.ClientCo
 
 func (d *deployCheckTestSuite) SetupTest() {
 	d.defaultDeploymentCheckCommand = deploymentCheckCommand{
-		file:               "testdata/deployment.yaml",
+		files:              []string{"testdata/deployment.yaml"},
 		retryDelay:         3,
 		retryCount:         3,
 		timeout:            1 * time.Minute,
 		printAllViolations: true,
 	}
 }
 
+func (d *deployCheckTestSuite) TestNormalizeYaml() {
```

Comment:
Just as a comment, feel free to ignore: I usually prefer using backsticks compared to double quotes for multiine strings, this makes things more readable to me.

Reply:
I prefer it as is